pkg/logs/sender/batch_strategy.go

```diff
@@ -134,6 +134,8 @@ func (s *batchStrategy) flushBuffer(outputChan chan *message.Payload) {
 
 func (s *batchStrategy) sendMessages(messages []*message.Message, outputChan chan *message.Payload) {
 	serializedMessage := s.serializer.Serialize(messages)
+	log.Debugf("Send messages (msg_count:%d, content_size=%d, avg_msg_size=%.2f)", len(messages), len(serializedMessage), float64(len(serializedMessage))/float64(len(messages)))
```

Comment:
@remeh Indeed, could make sense as Trace level too.

The rational for having it at debug level is that the number of messages shouldn't be that high since the log will print each time for a batched messages, for many EP track, it should be only trigger once every 5sec (https://github.com/DataDog/datadog-agent/blob/c4bf7015c8ddb9f27be0776b8e168fea5bccd3e5/pkg/config/config.go#L50-L51) for 1 payload.